package.json

```diff
@@ -24,14 +28,14 @@
         "jest": "^23.6.0",
         "knex": "^0.16.3",
         "mock-knex": "^0.4.3",
-        "semantic-release": "^15.13.2",
         "ts-jest": "^23.10.5",
         "tslint": "^5.12.0",
         "tslint-config-prettier": "^1.17.0",
         "tslint-eslint-rules": "^5.4.0",
         "typescript": "^3.2.2"
     },
     "peerDependencies": {
-        "knex": "^0.16.0"
+        "knex": "^0.16.0",
+        "tslib": "^1.9.3"
```

Comment:
This could be a breaking change? Thoughts?

Reply:
Are we removing semantic-release? and what do you see as the breaking change?

Reply:
As a consumer, you will need to install tslib otherwise the code will break

tslib has the helper functions in it, rather than them being baked into every module

Reply:
Not gonna make it breaking